### DISCUSSION TITLE
More fields  in admin joomla backend view

### DISCUSSION THREAD

**Discussion by infotecnica** [joomengine/Joomla-Component-Builder#685] [General]
Hello everyone

JCB automatically adds some necessary fields, but these are not displayed in the component lists when I add items from the joomla backend.
How can I add the created field and others to this list?

**Comment by Llewellynvdm** [comment]
There is [a tutorial](https://github.com/vdm-io/Joomla-Component-Builder/wiki#how-to-overwrite-the-custom-fields), you can search for stuff in the wiki as it is all the tutorial in text form, which makes it easy to search like this:
https://github.com/vdm-io/Joomla-Component-Builder/search?q=%22default+fields+in+list+view%22&type=wikis

So JCB has improved since many of the tutorials were made, but things still work the same and in many cases better...

**Reply by infotecnica** [reply]
Thanks, I followed the tutorial and I think I understand how to add them
